docs/src/pages/getting-started/appdir.mdx

```diff
@@ -110,6 +110,10 @@ export const { UploadButton, UploadDropzone, Uploader } =
 
 ### Mount A Button And Upload!
 
+<Callout type="info">
+Don't forget to add the `"use client;"` directive at the top of your file, since `UploadButton`need to run on the client-side.
```

Comment:
```suggestion
Don't forget to add the `"use client;"` directive at the top of your file, since the `UploadButton` component needs to run on the client-side.
```